web/server.go

```diff
@@ -263,6 +263,9 @@ func dbInit(dbtype, dburi string) (*sql.DB, error) {
 	}
 	db.SetMaxOpenConns(Config.MaxDBConnections)
 	db.SetMaxIdleConns(Config.MaxIdleConnections)
+	if dbtype == "sqlite3" {
+		db.Exec("PRAGMA journal_mode=WAL;")
```

Comment:
Could you please add proper comment why do we need this option, and provide pointer to docs we found on google about it. It is good practice to make reference to features we borrow from someone else discussion